.github/workflows/create-prerelease-branch.yml

```diff
@@ -0,0 +1,109 @@
+          # actions, when using the default account, cannot trigger other actions. And we want this
+          # action to trigger the regular CI pipeline on the created branch.
+          # This machine account is only for this PAT, pwd was created and thrown away
+          # If any update needed, create a new account, add access to the repo and generate a new PAT
+          token: ${{ secrets.MACHINE_ACCOUNT_PAT }}
+
+      # Setup bot information for creating pull request
+      # Here we use the id from the github actions bot: https://api.github.com/users/better-informatics%5Bbot%5D
+      - name: Setup bot git information
+        run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+
+      # Check out onto desired branch or tag to create release from
+      - name: Checkout tag/branch
+        run: git checkout ${{ github.event.inputs.branch }}
+
+      # Ensure node version is great enough
+      - name: Use Node.js v14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      # Try get node_modules from cache
+      - name: Restore node_modules from cache
+        uses: actions/cache@v2
+        with:
+          path: common/temp/pnpm-store
+          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
+
+      # Bump package versions
+      - name: Bump beta package versions if beta
+        if: ${{ github.events.inputs.bump_type == 'beta' }}
+        run: node common/scripts/bump-beta-version.js
+
+      - name: Bump stable package versions if stable
+        if: ${{ github.events.inputs.bump_type == 'stable-minor' || 'stable-patch' }}
+        run: node common/scripts/bump-stable-version.js
+
+      # Generate changelog using beachball
+      - name: Generate changelog using beachball
+        run: node common/scripts/beachball-changelog-only.js
+
+      # Important to check version consistency again after bumping versions.
+      - name: Ensure all package versions are consistent
+        run: rush ensure-consistent-versions
+
+      # Retrieve new version to create branch with
+      - name: Retrieve new version from package.json
+        id: version
+        run: |
+          ver=$(jq -r .version packages/communication-react/package.json)
+          echo version: $ver
+          echo "::set-output name=version::$ver"
+
+      # Commit changes
+      - name: Hop into new branch
+        run: git checkout -b prerelease-${{github.events.inputs.bump_type}}/${{ steps.version.outputs.version }}
+      - name: Commit changes
+        run: |
+          git add .
+          git commit -m "${{github.events.inputs.bump_type}} version bump pre-release branch"
+
+      # create Groom branch for the changelog and version bumps
+      - name: Hop into Groom branch
+        run: git checkout -b groom_changelog_${{ steps.version.outputs.version }}
```

Comment:
Again, don't you need to push this branch before creating a PR?

Also, GitHub doesn't like PRs that contain no changes. You'll likely have to modify the CHANGELOGE.md file with a placeholder `Please Groom This File` on top or something so that GH will let you create the PR.